frontends/web/src/locales/en/app.json

```diff
@@ -577,6 +578,7 @@
     "appVersion": "App version:"
   },
   "generic": {
+    "and": "and",
```

Comment:
such tricks usually don't work out in my experience. there is a good chance some languages don't function this way. For example, in Japanese there is probably a comma between all coins (no "and").

If you have to include the coins in the list (see my other comment), it is safer to do sth like  "The exchange rates of the following coins are out of date: A, B, C, D".